### ISSUE TITLE
[a11y]: Chat bot drift isn't mobile friendly

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5630]
### Audit type

Mobile device navigation

### User journey audit issue

#5467

### Problem description

The Drift chat bot welcome message can't be exited on small mobile screens (`320px`)
<img width="320" alt="Screen Shot 2022-07-29 at 9 48 30 AM" src="https://user-images.githubusercontent.com/59381432/181802373-659460d3-d5e2-43a6-9ed4-4827fbb0c127.png">

### Expected behavior

All libraries used in our application should have mobile considerations down to `320px` screen width
